### PR TITLE
Bytecode file

### PR DESCRIPTION
**Pull request by rocky**

### CODE REVIEW

**Comment by rocky** [comment]
Although the information in the Bytecode Files section contains some information outside of that topic, I'm not sure how to reorganize things. I think this info should be in there somewhere and somewhere in the Environment section. 

I hadn't realized before that there is some duplication in the section on Emacs Lisp Bytecode Objects with parts of the Introduction. I'm not sure how to deal with that. I suppose a little bit of duplication is okay. In any event I've expanded on what you wrote before. 

Let me know what you think, especially if you have thoughts for how to reorganize.

File: environment.texi [hunk 1]
```diff
@@ -28,7 +46,7 @@ accepts Emacs Lisp input and produces bytecode that can be run by
 Emacs. The compiler itself is written in Emacs Lisp @footnote{usually,
 the compiler itself is compiled into bytecode, which avoids overflow
 problems}, and is a comparatively compact program contained in the
-files bytecomp.el and byte-opt.el.
+files @code{bytecomp.el} and @code{byte-opt.el}.
```
Comment (pipcet) [inline environment.texi:49]: We should probably mention `cconv.el`code

Comment (rocky) [reply environment.texi:49]: Sure. Please suggest something.

**Review by pipcet — APPROVED** [review]
Wow, this is a lot of work. Looks good to me!

**Comment by rocky** [comment]
Ok. We'll go with it and revise as needed down the line.

**Pull request merged**